testing/smoke/main.tf

```diff
@@ -10,6 +10,21 @@ terraform {
 
 provider "ec" {}
 
+module "tags" {
+  source  = "../infra/terraform/modules/tags"
+  project = "smoke-test"
```

Comment:
`smoke-test` does not give me any information valid to identify where the deployment come from, àpm-server` does

Reply:
see `repo`:
- https://github.com/elastic/apm-server/pull/15884/files#diff-1b0205a341d2e4aec28dde78c2d93be8fc938d7ac1feb9228e4b4bb1154a478fR21

Reply:
I said it, `repo` has the repository, I do not talk about that. I am saying that `project: smoke-test` means nothing. `smoke-test` is a kind of test is not a project, here the only project I see is `apm-server`

Reply:
The `repo: apm-server` and `project: smoke-tests` tags/labels are meaningful, IMO. `project` is quite generic and is in fact, what the APM Server team has done in other places:

- https://github.com/search?q=repo%3Aelastic%2Fapm-server+project+language%3AHCL&type=code&l=HCL

If it is changed here, I'd suggest changing it to other places - to help with the semantic

I do understand caching is a bit easier than naming things 👼